src/cljc/quil/core.cljc

```diff
@@ -4963,3 +4963,20 @@
      [orig-name]
      (let [res (docs/matching-processing-methods fn-metas orig-name)]
        (u/print-definition-list res))))
+
+
+#?(:cljs 
+    (defn preload-font
+      "Fonts can be preloaded from URL, such as \"Arial.ttf\".
+      Be careful! Font name must be the same in preload-font call and in create-font call.
+
+      Sample:
+      (q/preload-font \"ComicSans.ttf\")
+      (q/text-font 
+        (q/create-font \"ComicSans.ttf\" 14))
```

Comment:
This increase the gap between Processing.js docs and Quil.
Do you think that is good idea? Because we hide font loading process from user in this case.

So, what happened if user call ```create-font``` inside ```:draw``` handler with slow fonts cdn?
I think he will see strange artifacts.

P.S. Yes, fonts are cached inside Processing.js and we don't need to reload it.